src/components/LeftSidebar/LeftSidebar.vue

```diff
@@ -926,6 +985,16 @@ export default {
 
 }
 
+.clear-filter__button {
+	border-radius : var(--border-radius-pill);
+	overflow: hidden;
+}
+
+:deep(.empty-content) {
+	text-align: center;
+	margin-top: 10%;
```

Comment:
default appearance for empty content is to center it in the sidebar. maybe stick to that?

Reply:
For the leftSidebar .. idk it seemed too low especially that user is expecting something to show at the very top. Setting a default appearance for empty content can be tricky. For square-like shapes or horizontal rectangles, this default is okay but for vertical rectangles, IMO it really depends on the situation.